blog/jenkins_github_pull_request_builder.adoc

```diff
@@ -0,0 +1,94 @@
+= Exploring how to build from pull requests in Jenkins for continuous feedback
```

Comment:
i think it is good content to have - I would just point out this is about building JBoss Tools Eclipse plugins which does make things a bit more different than normal java projects in form of updatesites and somewhat larger dependency sets.